autoload/lsp/ui/vim/folding.vim

```diff
@@ -187,6 +187,12 @@ function! s:handle_fold_request(server, data) abort
     " 'foldexpr'. Only do this if the user hasn't changed 'foldmethod',
     " and this is the correct buffer.
     let l:current_window = winnr()
+    let l:oldmode = mode()
```

Comment:
Thanks your fix.

s/oldmode/old_mode/

Reply:
Thanks for your review.
I've fixed and force pushed.